operators/hpe-csi-operator/deploy/role.yaml

```diff
@@ -58,7 +58,7 @@ rules:
     verbs: ["get", "list", "watch"]
   - apiGroups: ["storage.k8s.io"]
     resources: ["volumeattachments"]
-    verbs: ["get", "list", "watch", "update", "patch"]
+    verbs: ["get", "list", "watch", "update", "patch", "delete"]
```

Comment:
Do we have this on all the yamls already?

Reply:
yes, i had added it with all other YAML's but missed with operator one.